### PR TITLE
Make awstpr open sourced under Apache License 2.0

### PR DESCRIPTION
**Pull request by nhlfr**
Provide necessary files to clarify the license of the project, maintainers, ways to contact with the community and the goals.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# How to contribute
+
+PROJECT is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers and other resources to make getting your contribution into PROJECT easier.
```
Comment (iaguis) [inline CONTRIBUTING.md]: s/PROJECT/awstpr/

Comment (iaguis) [inline CONTRIBUTING.md]: Same for all instances of PROJECT

File: README.md [hunk 2]
```diff
@@ -1,3 +1,45 @@
...
+Specification of the third party objects used to deploy Kubernetes on top of AWS by the "undercloud" Kubernetes running
+[aws-operator](https://github.com/giantswarm/aws-operator).
+
+## Getting Project
+
+Download the latest release: https://github.com/giantswarm/awstpr/releases/latest
+
+Clone the git repository: https://github.com/giantswarm/awstpr.git
+
+Download the latest docker image from here: https://hub.docker.com/r/giantswarm/awstpr/
```
Comment (iaguis) [inline README.md]: Not sure we'll provide docker images, maybe we should remove that

File: README.md [hunk 3]
```diff
@@ -1,3 +1,45 @@
 [![CircleCI](https://circleci.com/gh/giantswarm/awstpr.svg?&style=shield&circle-token=752d6ec42799fb9fd56dcf13c142d2f675b0b91f)](https://circleci.com/gh/giantswarm/awstpr)
 
 # awstpr
+
+Specification of the third party objects used to deploy Kubernetes on top of AWS by the "undercloud" Kubernetes running
+[aws-operator](https://github.com/giantswarm/aws-operator).
+
+## Getting Project
+
+Download the latest release: https://github.com/giantswarm/awstpr/releases/latest
```
Comment (iaguis) [inline README.md]: Not sure what the release artifacts would be

File: README.md [hunk 4]
```diff
@@ -1,3 +1,45 @@
...
+- IRC: #[giantswarm](irc://irc.freenode.org:6667/#giantswarm) on freenode.org
+- Bugs: [issues](https://github.com/giantswarm/PROJECT/issues)
+
+## Contributing & Reporting Bugs
+
+See [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches, the contribution workflow as well as reporting bugs.
+
+## License
+
+PROJECT is under the Apache 2.0 license. See the [LICENSE](LICENSE) file for details.
```
Comment (iaguis) [inline README.md]: s/PROCECT/awstpr/

File: README.md (hunk 3 above)
Comment (nhlfr) [inline README.md]: Stable versions of specs we would use for stable versions of aws-operator?

**Comment by nhlfr** [comment]
@iaguis PTAL again

File: README.md (hunk 3 above)
Comment (iaguis) [inline README.md]: Yeah, but I expect that to be on git tags, not github releases. We can change this to:

```
Check out the latest tag: https://github.com/giantswarm/awstpr/tags
```

**Comment by nhlfr** [comment]
@iaguis done

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+# Email and chat
+
+- Email: [giantswarm](https://groups.google.com/forum/#!forum/giantswarm)
+- IRC: #[giantswarm](irc://irc.freenode.org:6667/#giantswarm) IRC channel on freenode.org
+
+## Getting started
+
+- Fork the repository on GitHub
+- Read the [README.md](https://github.com/giantswarm/example-opensource-repo/blob/master/README.md) for build instructions
```
Comment (iaguis) [inline CONTRIBUTING.md]: This link should go to `https://github.com/giantswarm/awstpr/blob/master/README.md`, not to the example

File: MAINTAINERS [hunk 6]
```diff
@@ -0,0 +1,2 @@
+Iago López Galeiras <iago@kinvolk.io> (@iaguis) pkg:*
+Michal Rostecki <michal@kinvolk.io> (@nhlfr) pkg:*
```
Comment (iaguis) [inline MAINTAINERS]: Don't you wanna use a `ł`? :P

File: CONTRIBUTING.md (hunk 5 above)
Comment (iaguis) [inline CONTRIBUTING.md]: Maybe just `[README](README.md)`

File: MAINTAINERS (hunk 6 above)
Comment (nhlfr) [inline MAINTAINERS]: https://www.youtube.com/watch?v=WWaLxFIVX1s :P

**Comment by iaguis** [comment]
lgtm

**Pull request merged**